utilities/Import-UniApimApi.ps1

```diff
@@ -0,0 +1,73 @@
+function Import-UniApimApi 
+{
+<#
+.SYNOPSIS
+Imports an API onto Azure API Management.
+
+.DESCRIPTION
+Imports an API onto Azure API Management.
+
+.PARAMETER Environment
+The Azure APIM environment.
+
+.PARAMETER ApiName
+The name of the API.
+
+.PARAMETER ApiPath
+The path of the API. Without the leading slash.
+
+.PARAMETER SpecificationPath
+The path to the API specification. Without the leading slash.
+
+.EXAMPLE
+Import-UniApimApi -Environment "dev" -ApiName "my-api" -ApiPath "my-api" -SpecificationPath "my-api/swagger.json"
+
+.NOTES
+At the moment this function only supports scenarios where the following is true:
+- The api is hosted under the "api" subdomain of one of the uniphar.ie zones.
+- The API is imported as an OpenAPI specification.
+- The API is imported to the "Development" product.
+#>
+    [CmdletBinding()]
+    param (
+        [parameter(Mandatory = $true, Position = 0)]
+        [ValidateSet('dev', 'test', 'prod')]
+        [string]$Environment,
+
+        [parameter(Mandatory = $true, Position = 1)]
+        [string]$ApiName,
+
+        [parameter(Mandatory = $true, Position = 2)]
+        [ValidatePattern('^[^/]')] # No leading slash
+        [string]$ApiPath,
+
+        [parameter(Mandatory = $true, Position = 3)]
+        [ValidatePattern('^[^/]')] # No leading slash
+        [string]$SpecificationPath
```

Comment:
I would call it "OpenApiSpecPath" but it might make a lot of sense if you are working here